packages/ros2/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(soss-ros2)
 find_package(soss-core REQUIRED)
 find_package(soss-rosidl REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(yaml-cpp REQUIRED)
```

Comment:
As mentioned for other packages, I believe this can be eliminated by adding `find_dependency(yaml-cpp)` to `soss-core`'s config file.